### PR TITLE
Remove native api and add linuxnative as separate helper

### PR DESCRIPTION
**Pull request by npmitche**

### CODE REVIEW

File: chipsec/helper/linuxnative/Makefile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+clean:
```
Comment (BrentHoltsclaw) [inline chipsec/helper/linuxnative/Makefile:1]: I don't think makefile is needed for native

File: chipsec/helper/linuxnative/linuxnativehelper.py [hunk 2]
```diff
@@ -0,0 +1,462 @@
...
+                    value = value & 0xFF
+                elif size == 2:
+                    value = value & 0xFFFF
+                elif size == 4:
+                    value = value & 0xFFFFFFFF
+
+                return value
+            else:
+                byte = b"\xff"
+                return defines.unpack1(byte * size, size)
```
Comment (BrentHoltsclaw) [inline chipsec/helper/linuxnative/linuxnativehelper.py:197]: Will this lead to false results?

File: chipsec/helper/linuxnative/linuxnativehelper.py [hunk 3]
```diff
@@ -0,0 +1,462 @@
...
+        if self.devport_available():
+            os.lseek(self.dev_port, io_port, os.SEEK_SET)
+            if 1 == size:
+                fmt = 'B'
+            elif 2 == size:
+                fmt = 'H'
+            elif 4 == size:
+                fmt = 'I'
+            else:
+                raise ValueError("Invalid size")
```
Comment (BrentHoltsclaw) [inline chipsec/helper/linuxnative/linuxnativehelper.py:341]: read condition does not have an else case that raises ValueError, suggest making it consistent

**Comment by BrentHoltsclaw** [comment]
May also need to look into:
exceptions.py
```
class UnimplementedNativeAPIError (UnimplementedAPIError):
    def __init__(self, api_name):
        super(UnimplementedNativeAPIError, self).__init__(api_name)
```

acpi.py
`get_ACPI_table_list` and `get_ACPI_table`

File: chipsec/helper/linuxnative/linuxnativehelper.py (hunk 2 above)
Comment (npmitche) [reply chipsec/helper/linuxnative/linuxnativehelper.py:197]: Would it be better to raise an exception in this case?

File: chipsec/helper/linuxnative/linuxnativehelper.py (hunk 3 above)
Comment (npmitche) [reply chipsec/helper/linuxnative/linuxnativehelper.py:341]: Read condition for io_port (`read_io_port:318`) does have similar condition. Were you referring to the `read_pci_reg` function?

**Review by dscott90 — APPROVED** [review]

**Review by Sae86 — APPROVED** [review]

**Pull request merged**